pkg/minikube/machine/client.go

```diff
@@ -95,7 +98,7 @@ func (api *LocalClient) NewHost(drvName string, rawDriver []byte) (*host.Host, e
 	if def.Init == nil {
 		return api.legacyClient.NewHost(drvName, rawDriver)
```

Comment:
add a todo with Issue number to clean up after we merge Lib Machine being migrated into minikube